src/main/resources/assets/README.md

```diff
@@ -23,14 +24,34 @@ easy editing, set the `assets_path` flag when running Marathon:
 4. Run development environment
 
         npm run watch
+
 5. Build the assets
 
         npm run dist
+
 6. The distribution files will be written to `src/main/resources/assets/dist`.
    Check it in.
 
         git add dist/main.js dist/main.css img/*
 
+#### Adding npm package dependencies to package.json
+
+If you want to add a new npm package to 'node_modules':
+
+1. Install the new package
+
+        npm install [your package] --save
+    will install and save to dependencies in package.json and
+
+        npm install [your package] --save-dev
+    will add it to devDependencies.
+
+2. Create a synced npm-shrinkwrap.json
+
+        npm shrinkwrap
```

Comment:
Missed this the first time. You need to run `npm shrinkwrap --dev` to get the devDependencies shrinkwrapped too.